pom.xml

```diff
@@ -232,6 +232,7 @@
         <okhttp.version>2.7.5</okhttp.version>
         <opensaml.version>3.4.3</opensaml.version>
         <pac4j.version>3.8.3</pac4j.version>
+        <powermock.version>2.0.2</powermock.version>
```

Comment:
Looks like 2.0.4 is out?

https://mvnrepository.com/artifact/org.powermock/powermock-core/2.0.4

Reply:
Done.